src/main/js/component/audit/Index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { BootstrapTable, ButtonGroup, TableHeaderColumn } from 'react-bootstrap-table';
-import { getAuditData } from 'store/actions/audit';
+import { getAuditData, resendNotification } from 'store/actions/audit';
```

Comment:
Why was this added?  Was it missing from Index.js beforehand and causing an error in the javascript console?

Reply:
Yes it was missing before

Reply:
This was because of the issue with My and Gavin's save actions. So I added it as I noticed it was broken when trying to test this feature.